src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Statistics {
+  // Contains sentiment scores in the range [-1, 1] with 0.2 intervals.
+  private Map<Range, Integer> aggregateValues;
+  private double averageScore;
+  private static final double INTERVAL = 1.0;
```

Comment:
You're setting interval to 1.0; however, before, when it was hard-coded, it was 0.2.